ui/app/src/components/PathNavigator/PathNavigator.tsx

```diff
@@ -331,7 +331,7 @@ export function PathNavigator(props: PathNavigatorProps) {
     if (type === 'language') {
       const locales = siteLocales.localeCodes?.map((code) => ({
         id: `locale.${code}`,
-        label: formatMessage(languages[code.toLowerCase()])
+        label: languages[code.toLowerCase()]
       }));
       setWidgetMenu({
```

Comment:
Is there currently a way of using this? I'm thinking we should clean out this code and avoid the languages to even be bundled if they're not in use.

Reply:
there's no way of using this currently, so yeah, I can remove this languages usage